scripts/main/settings.js

```diff
@@ -483,3 +483,47 @@ settings.save_enter = function (e) {
 		},
 	});
 };
+
+settings.viewToken = function () {
+	api.post("User::getCurrent", {}, function (data) {
```

Comment:
Please annotate `data` with the correct jsDoc type as done everywhere else.

Reply:
It is possible. I did that at a lot of places. You simply need to write the annotation in front of the lambda between the preceding comma `,` and the keyword `function` like this

```js
api.post("User::getCurrent", {}, /** here goes the annotation */ function (data) {
```

For better readability you can use line breaks

```js
api.post(
	"User::getCurrent",
	{},
	/**
	 * here goes the annotation
	 */
	function (data) {
```

Or the third alternative is to pull out the lambda, define it as a proper method like

```js
/**
 * here goes the annotation
 */
const getAuthenticatedUserHandler = function (data) {
}
```

But not annotating the parameter is not an option for me.